exampleCourse/questions/demo/overlayDropdown/question.html

```diff
@@ -0,0 +1,19 @@
+<pl-overlay width="700" height="700" clip="false">
+    <pl-background>
+        <img src="{{options.client_files_question_url}}/europe_annotated.png" style="max-width: 100%; max-height: 100%">
```

Comment:
Can you put in a sentence or two at the top explaining that this is a question using the `pl-overlay` and `pl-dropdown` elements? You can probably reuse most of the text from the other `element/overlay` question.

Reply:
Alright. This is how it looks now:
![image](https://user-images.githubusercontent.com/7872295/85879169-bbaae680-b78e-11ea-84ae-00c9dbe40774.png)

Reply:
*Fixing the "Writing the name..." issue.

Reply:
Fixed. Is now "Select the names of the unlabeled countries:"